.github/workflows/cd.yml

```diff
@@ -0,0 +1,22 @@
+name: CD
+on:
+  push:
+    tags:
+      - v*  
+jobs:
+  cd:
+    name: CD
+    runs-on: ubuntu-latest
+    environment: CD
```

Comment:
This requires to move the `NPM_TOKEN` secret from a Repo Secret to an Environment Secret on the Repo.

Reply:
Named environment comes with more safe-guards:
- you can impose a delay between the trigger and the actual run
- you can request a human review before an environment is used
- **the one that would be of interest to us**: you can limit the use of the environment on specific branches

So this allow/prevent accidental or malicious publishing from an unreviewed/unprotected branch